src/server.js

```diff
@@ -53,7 +53,7 @@ export class Server {
     submitTransaction(transaction) {
         let tx = encodeURIComponent(transaction.toEnvelope().toXDR().toString("base64"));
         var promise = axios.post(
-              URI(this.serverURL).path('transactions').toString(),
+              URI(this.serverURL).segment('transactions').toString(),
```

Comment:
Can you add tests to this? I think it's working as expected. Looking at [docs](https://medialize.github.io/URI.js/docs.html#accessors-segment), `segment` does rewrite the full path and not append a new segment to it.

Reply:
The urijs docs show examples where:
urijs.segment(string) appends, 
urijs.segment(array) sets, 
and urijs.path(string) sets.

urijs.path(string) didn't work for horizon server path with folder paths.
I've added some unit tests.
Please review.